palm/plugins/dbt/dbt_containerizer.py

```diff
@@ -133,11 +68,25 @@ def replacements(self) -> dict:
         """
         Return a dictionary of replacements for the dbt template.
         """
-        return {
+        profile_strategy = self.determine_profile_strategy(Path.cwd())
```

Comment:
destructuring the returned tuple here might make the use of `profile_strategy` more clear below?

```
profile_host, profile_volume  = self.determine_profile_strategy(Path.cwd())
```

Reply:
yeah good call as well

Reply:
updated to instance prop.